From 05e5ebaabaec7ebd9737a7be17035b07c4ecafd9 Mon Sep 17 00:00:00 2001
From: ethan <ethangoh98@gmail.com>
Date: Mon, 2 Dec 2024 00:55:32 +1100
Subject: [PATCH] adding libms_pen_charger package

---
 base.mk | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/base.mk b/base.mk
index b6ea182..8bd3775 100644
--- a/base.mk
+++ b/base.mk
@@ -281,6 +281,7 @@ PRODUCT_PACKAGES += \
 	SurfaceDuo2PostureProcessorOverlay \
 	SurfaceDuoSettingsOverlay \
 	SurfaceDuoSettingsIconsOverlay \
-	SurfaceDuoAppHintingOverlay
+	SurfaceDuoAppHintingOverlay \
+	libms_pen_charger
 
 $(call inherit-product-if-exists, vendor/surface/prebuilts/bootanimation/bootanimation.mk)
\ No newline at end of file
-- 
2.43.0

